app/views/my/password.html.erb

```diff
@@ -26,16 +26,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-
 <% html_title l(:label_my_account), l(:button_change_password) %>
 <%= toolbar title: l(:button_change_password) %>
 
 <%= error_messages_for 'user' %>
 <%= styled_form_tag({:action => :change_password},
-                    {:autocomplete => 'off'}) do %>
+                    {:autocomplete => 'off', class: 'change_password large-5 columns -higher'}) do %>
```

Comment:
@floriank I'm not familiar with all this stuff, I solved to problem with frontend knowledge that I have